aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +261,45 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Forms in layers vs. full page 
+
+A full-page form is used to convey information without the possibility of several interpretations. Depending on the type of form we are designing, a well-structured form on how and when to use forms will guide users with minimal dispute. 
+
+**Best Practices**
+
+- When using a full-page form use a single-column form layout to create a clear focused path to complete for the user.
+- When using a full-page form avoid using Multi-column layouts not to cause users to skip important fields, input data in the wrong field, or simply hinder efficiency which leads to frustration or even abandonment.
+- When using a single form-page use no more than 10 form fields in the same viewport so that users are not intimidated by seeing a long page filled with form fields and sections.
+- Use [center modals](https://design-system.hpe.design/components/layer#center-modal) for full-page forms to display information without leaving the current contexts. 
```

Comment:
Wondering if it would be clearer to separate this list into two sections:

Best practices for full-page forms:
- Use a single-column...
- Avoid using multi-column..

Best practices for forms in layers:
- Use center modals...
- Use side drawer...
...

Reply:
good point I agree would be easier to scan

Reply:
good point, i can update the figma